src/Compiler/Microsoft.AspNetCore.Mvc.Razor.Extensions.Version1_X/src/ViewComponentTagHelperDescriptorFactory.cs

```diff
@@ -58,7 +58,11 @@ public virtual TagHelperDescriptor CreateDescriptor(INamedTypeSymbol type)
         var tagName = $"vc:{HtmlConventions.ToHtmlCase(shortName)}";
         var typeName = $"__Generated__{shortName}ViewComponentTagHelper";
         var displayName = shortName + "ViewComponentTagHelper";
-        var descriptorBuilder = TagHelperDescriptorBuilder.Create(ViewComponentTagHelperConventions.Kind, typeName, assemblyName);
+
+        using var _ = TagHelperDescriptorBuilder.GetPooledInstance(
```

Comment:
Maybe after we make `_` always mean discard we actually implement this feature...

Reply:
Yeah, we need to be able to write `using _ = ...`. +100